### DISCUSSION TITLE
How do i change the money system to {cash::%player's uuid%}

### DISCUSSION THREAD

**Discussion by DumpyMc** [BG-Software-LLC/SuperiorSkyblock2#1753] [Approved Feature Requests] [answered]
So i have my own economy how do i change your economy to my economy. 

What im trying to say is how would i switch my economy over to yours?
Im struggling hard

**Comment by DumpyMc** [comment]
this is about the plugin Superiorskyblock2

**Comment by OmerBenGera** [comment]
You can do that using the API and register a custom EconomyProvider (Implement your own one and then register it)

**Reply by DumpyMc** [reply]
how would i register it?

**Reply by DumpyMc** [reply]
btw good plugin just letting yk incase no one else says anything lol

**Reply by DumpyMc** [reply]
Would i do that here with the public interface EconomyProvider {

**Comment by OmerBenGera** [comment]
Implement your own EconomyProvider in your plugin.
Then, when your plugin is enabled, check if SSB is enabled. If so, use SuperiorSkyblockAPI.getProviders#setEconomyProvider
This will make it so your custom economy provider will be used for warps charges, upgrades, etc.
For bank interactions (deposit or withdraw), use setBankEconomyProvider instead.

**Comment by DumpyMc** [comment]
i feel stupid how would i do that cause im also on xbox would i go to the plugin on minehut then go to upgrades or what or if i send you the cash uuid can you do it for me and send the file pls

**Reply by DumpyMc** [reply]
This is the uuid for my economy %Format({cash::%uuid of player%})%
if you can do it pls do lol

**Reply by DumpyMc** [reply, marked as answer]
i found out how to do it

**Comment by DumpyMc** [comment]
Where would i go to do that would i edit that actual file before installing it on my server? Or would i do it when it is installed?


    On Friday, June 23, 2023 at 05:12:14 AM PDT, Ome_R ***@***.***> wrote:  
 
 


Implement your own EconomyProvider in your plugin.
Then, when your plugin is enabled, check if SSB is enabled. If so, use SuperiorSkyblockAPI.getProviders#setEconomyProvider
This will make it so your custom economy provider will be used for warps charges, upgrades, etc.
For bank interactions (deposit or withdraw), use setBankEconomyProvider instead.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
